oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/query/ElasticRequestHandler.java

```diff
@@ -445,6 +445,12 @@ public PhraseSuggester suggestQuery() {
         return PhraseSuggester.of(ps -> ps
                 .field(FieldNames.SPELLCHECK)
                 .size(10)
+                // The Elasticsearch Java client fails parsing a response to suggest queries if the highlight is not set.
+                // Caused by: co.elastic.clients.util.MissingRequiredPropertyException: Missing required property 'PhraseSuggestOption.highlighted'
+                //	at co.elastic.clients.util.ApiTypeHelper.requireNonNull(ApiTypeHelper.java:76)
+                //	at co.elastic.clients.elasticsearch.core.search.PhraseSuggestOption.<init>(PhraseSuggestOption.java:64)
+                // Happens with Elasticsearch server 8.4.1 and client 7.17.6
+                .highlight(f -> f.preTag("").postTag(""))
```

Comment:
Is there already an open issue on that? In that case, can we link it? Otherwise we should create one.

Reply:
I opened this issue: https://github.com/elastic/elasticsearch-java/issues/404